### PR TITLE
SOV-1025: feat: add mobile header

### PR DESCRIPTION
**Pull request by creed-victor**
https://sovryn.atlassian.net/browse/SOV-1025

### CODE REVIEW

File: apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -36,17 +41,36 @@ export const ConnectWalletButton: FC<
...
+          onDisconnect={onDisconnect}
+          address={address}
+          dataLayoutId={dataLayoutId}
+          className={className}
+          balance={
+            <Menu className="mb-4">
+              <MenuItem
+                text={t(translations.connectWalletButton.rewards)}
+                className="no-underline"
+                href="https://live.sovryn.app/reward"
```
Comment (pietro-maximoff) [inline apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx]: replace https://live.sovryn.app/ with {sovrynLinks.dappAlpha}

**Review by pietro-maximoff — CHANGES_REQUESTED** [review]
@Victor added one comment, other stuff look good

**Review by pietro-maximoff — APPROVED** [review]

File: apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx [hunk 2]
```diff
@@ -36,17 +42,36 @@ export const ConnectWalletButton: FC<
...
-          disconnect: 'Disconnect',
-        }}
-      />
+      <>
+        <WalletIdentity
+          onDisconnect={onDisconnect}
+          address={address}
+          dataLayoutId={dataLayoutId}
+          className={className}
+          balance={
```
Comment (tiltom) [inline apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx]: This is weird, can you confirm with Pili if we don't want to have balance for the first iteration and if so, rename the prop appropriately? It just seems weird to assign all of the links to `balance`.

File: apps/frontend/src/app/2_molecules/NavLink/NavLink.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            {
+              [styles.active]: isActive,
+            },
+            className,
+          )
+        }
+      />
+    </li>
+  );
+};
```
Comment (tiltom) [inline apps/frontend/src/app/2_molecules/NavLink/NavLink.tsx]:
```suggestion
export const NavLink: FC<NavLinkProps> = ({ className, ...props }) => (
  <li>
    <BaseNavLink
      {...props}
      className={({ isActive }) =>
        classNames(
          styles.navLink,
          {
            [styles.active]: isActive,
          },
          className,
        )
      }
    />
  </li>
);
```

**Review by tiltom — APPROVED** [review]
Looking good, there are just 2 small changes to be made that don't block testing (but fix them pls :D ).

File: apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx (hunk 2 above)
Comment (creed-victor) [reply apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx]: yep, it was confirmed with phili that we will not show balance anymore, at least at first. I will change prop name to something that could contain any kind of stuff, maybe `content`

**Review by LunaSov — APPROVED** [review]

**Pull request merged**